include/SDL3/SDL_gpu.h

```diff
@@ -159,6 +159,24 @@
  * and reasonable to implement across several platforms and underlying APIs.
  * So while these things are not in the "never" category, they are definitely
  * not "near future" items either.
+ *
+ * ## System Requirements
+ *
+ * **Vulkan:** Supported on Windows, Linux, and Nintendo Switch. Requires Vulkan 1.0 with the following extensions and device features:
```

Comment:
Maybe also mention "certain Android devices" here?

Reply:
Oh gosh I completely erased Android from my mind... will fix, thanks

Reply:
Believe me, I would like to as well!